tests/Fixtures/ObjectWithNullObject.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\Serializer\Tests\Fixtures;
+
+use JMS\Serializer\Annotation\DeserializeNull;
+use JMS\Serializer\Annotation\Type;
+
+class ObjectWithNullObject
+{
+    /**
+     * @var null
+     * @Type("NullObject")
+     * @DeserializeNull()
```

Comment:
The annotation `DeserializeNull` is fine IMO, but what if we add also `SerializeNull` when doing the serialization.. can we find a name for a single annotation that allows to decide how nulls are handled, both on serialization and deserialization?

Reply:
Hm, the only thing that pops into my mind is:

```
@NullValue(serialize=false, deserialize=false)
```

Reply:
Another question. What happens if `NullObject` is an exiting class and there is no handler for it? 

From what I can see, 
```
{
 "nullProperty": null
}
```
When unserialized, nullProperty  will be an object instance of `NullObject`, and that should not happen